tests/integration/testdata/invoke/main.py

```diff
@@ -60,3 +60,10 @@ def execute_git(event, context):
 
 def no_response(event, context):
     print("lambda called")
+
+
+def image_handler(event, context):
+    f = open("image-for-lambda.png", "rb")
+    image_bytes = f.read()
+    f.close()
```

Comment:
you can open a file with context which should close it automatically
```suggestion
    with open("image-for-lambda.png", "rb") as f:
        image_bytes = f.read()
```